packages/11ty/_plugins/figures/transform.js

```diff
@@ -9,24 +9,26 @@ const { info, error } = chalkFactory('plugins:iiif:createImage')
  * @param  {Object} config Quire IIIF Process config
  * @return {Function}      createImage()
  */
-module.exports = (eleventyConfig) => {
+module.exports = (iiifConfig, figure, options={}) => {
   const {
     formats,
+    imageTransformations,
     inputDir,
     inputRoot,
     outputDir,
     outputRoot 
-  } = eleventyConfig.globalData.iiifConfig
+  } = iiifConfig
+
+  const transform = async (transformation) => {
+    /**
+     * Creates an image in the output directory with the name `${name}${ext}`
+     *
+     * @param  {Object} figure Figure entry data from `figures.yaml`
+     * @param  {Object} transformation A transformation item from `iiif/config.js#imageTransformations`
+     * @property  {String} name The name of the file
+     * @property  {Object} resize Resize options for `sharp`
+     */
```

Comment:
Should this be the following?
```suggestion
  /**
   * Creates a `sharp/transform` that writes the image file in the output directory
   * Nota bene: this `transform` is distinct form the `11ty/transform`
   *
   * @param  {Object} transformation A transformation item from `iiif/config.js#imageTransformations`
   * @property  {String} name The name of the file
   * @property  {Object} resize Resize options for `sharp`
   */
  const transform = async (transformation) => {
```